static/.well-known/assetlinks.json

```
@@ -0,0 +1,14 @@
+[
+    {
+        "relation": [
+            "delegate_permission/common.handle_all_urls"
+        ],
+        "target": {
+            "namespace": "android_app",
+            "package_name": "org.openlibrary.testing.twa",
```

We'll have to regenerate the apk to use this package_name
```suggestion
            "package_name": "org.openlibrary.twa",
```

---

Done.
To do: Create a new APK pointing to testing.